src/main/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>BATTLESHIP</title>
+    <link rel="stylesheet" href="../../src/css/main.css">
```

Comment:
Styles should be imported in the `JS` file, not in the `HTML` view file. Move it there.